force-app/main/educationCloudSettings/lwc/edcSettingsProductCard/edcSettingsProductCard.html

```diff
@@ -17,38 +17,41 @@ <h3 slot="title" id="cardTitle">
                     </div>
                 </h3>
                 <!-- Card Body -->
-                <div class="slds-grid slds-grid_vertical">
+                <div class="slds-grid slds-wrap">
                     <div class="slds-col slds-var-m-horizontal_small productDescription"><p>{description}</p></div>
                     <div class="slds-col slds-var-m-horizontal_small slds-var-m-top_small">
-                        <lightning-button-group>
-                            <lightning-button
-                                if:true={settingsComponent}
-                                label={labelReference.settingsButton}
-                                title={settingsButtonA11y}
-                                aria-described-by="cardTitle"
-                                onclick={handleSettingsClick}
-                                class="btnSettingsComponent"
```

Comment:
I removed the "class" properties on these 3 lightning-buttons because they weren't being used: there were no styles defined for the class names.

Reply:
@eclemmer the classes are needed for the jest test methods. In fact, it seems the jest test failed. In order to fix them I believe we need to add this back. You can run the jest test classes locally if you want to make sure they pass. Run 'yarn test:unit --coverage' in your terminal.

Reply:
@salvadorsfdc Thanks!  I have restored those.

Reply:
awesome! thanks